pkg/logs/input/file/file_provider.go

```diff
@@ -71,9 +73,11 @@ func (p *Provider) FilesToTail(sources []*config.LogSource) []*File {
 
 	if len(filesToTail) == p.filesLimit {
 		log.Warn("Reached the limit on the maximum number of files in use: ", p.filesLimit)
+		status.Warnings.RaisedWarnings[status.OpenFilesLimitReachedType] = true
```

Comment:
I think if the interface of `Warnings` was `status.Warnings.Raise(key string, message string)`, we could add any kind of message